public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Eine Alternative zu dem Nix Ökosystem",
+
+	"header.community": "Gemeinschaft",
+
+	"goals.title": "Die Ziele",
+	"goals.independent.title": "Unabhängigkeit",
+	"goals.independent": "Wir werden Nix, NixPkgs und NixOS forken und betreuen. Zusätzliche Unterstützung für Projekte wie Nix Darwin und Home Manager wird in Betracht gezogen.",
+	"goals.gov.title": "Gesellschaftliche Organisation",
+	"goals.gov": "Zur Verwaltung des Projekts wird ein demokratisches Modell mit gewählten Positionen eingesetzt. Ein Lenkungsausschuss (steering committee) entscheidet über den Kurs, während weitere Ausschüsse spezifische oder logistische Aufgaben übernehmen. Features oder einzelne Projekte werden von Special Interest Groups umgesetzt. SIGs werden in Arbeitsgruppen (Working Groups) zusammenarbeiten, um ihre Ziele übergreifend zu erreichen. Für jedes Komitee und jede Gruppe sind die Ziele, Anforderungen und Managementprozesse klar definiert.",
+	"goals.stabilization.title": "Stabilisierung",
+	"goals.stabilization": "Benutzerfreundliche Funktionen wie die 'Nix v3 CLI' werden standardmäßig aktiviert. Nix 'Flakes' werden bereits zu häufig verwendet, um sie zu entfernen oder zu ändern, stattdessen werden diese als 'flake v0' in ihrem aktuellen Zustand vereinheitlicht. Zukünftige Arbeiten werden von einer Special Interest Group gesteuert.",
+	"goals.infra.title": "Infrastruktur",
+	"goals.infra": "Wir werden die Tools und die Infrastruktur aufbauen und betreuen, welche erforderlich sind, um das Ökosystem stabil zu halten, einschließlich 'Continous Integration' und 'Binary Caching'.",
+	"goals.education.title": "Dokumentation",
+	"goals.education": "Einheitliche Dokumentation, Anleitungen, Leitfäden und Beispiele sorgen für ein besseres Onboarding-Erlebnis für Neulinge und einen schnelleren Iterationszyklus für Experten.",
+
+	"roadmap.title": "Der Fahrplan",
```

Comment:
I'd just leave it as "Die Roadmap", at least in technology roadmap is a pretty established term, even has a [Duden entry](https://www.duden.de/rechtschreibung/Roadmap).
"Fahrplan" is mostly associated with transport, trains etc.

Reply:
It's a slightly hidden nod to the german hacker scene that actually uses the term quite regularly for such things
https://events.ccc.de/tag/fahrplan/

I didn't see the Duden entry for that, nice catch. If above isn't resonating with you I'm happy to change.

Reply:
I like the idea, but I tend to agree with @christoph-heiss - in this case 'Fahrplan' being associated with event planning, and auxolotl hopefully not being 'just' a short event :P 

But I'm curious what @christoph-heiss thinks

Reply:
'Roadmap' it is.